### ISSUE TITLE
Excel column headings not announced if cells are locked 

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#6458]
If a row is set as column headings in Excel and those cells are then locked, NVDA does not announce the headings when moving across following rows.  I have not checked row headings, but anticipate the same issue would apply.  JAWS is able to announce column headings when cells are locked.
Andrew

**Comment by jcsteh** [comment]
Duplicate of #6000. Unfortunately, for some unknown reason, Excel won't allow us to retrieve cell region information in protected sheets. We need cell region information to be able to accurately retrieve row/column headers. As for JAWS, it's possible JAWS doesn't consider actual Excel regions (in contrast to JAWS-defined regions) when determining headers. If this is true, this is problematic because regions frequently have different headers from other regions. See also #4409.

**Comment by andrew-l-d** [comment]
Apologies.  My search did not pick up either #6000 or #4409. More 
embarrassing is that I created #4409.


Andrew
